assets/components/widgets/autoDirector/autoDirector.jsx

```diff
@@ -149,8 +149,8 @@ UI.scoringRules = {
 			return score;
 		}
 
-		var secondsPassed = ((UI.state.sessionInfo.timeTotal/3600)*60)-UI.state.sessionInfo.timeLeft
-		if (secondsPassed < 20 && driver.scoreInfo.positionOverall === 4) {
+		var secondsPassed = UI.state.sessionInfo.timeTotal;
+		if (secondsPassed < 20 && driver.scoreInfo.positionOverall === 1) {
```

Comment:
Should this still be 4? This code covers the race start, so we want to pick a driver not at the start, so it shows more action. ie. If we pick driver 1, then we miss some action.